### PR TITLE
Integrate swift lint + fix warnings

### PR DESCRIPTION
**Pull request by yageek**
I integrated `swiftlint` to the project. I disabled all rules that immedialty produced an error : 

- large_tuple
- force_cast
- todo
- nesting
- lin_length configured bigger as default

Then I fixed all the warnings related to `swiftlint` + one more regaring one deprecated SFSymbol image name

### CODE REVIEW

File: ViteMaDose/Views/CentresList/Cells/CentreCell.xib [hunk 1]
```diff
@@ -157,7 +157,7 @@
                                                     <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="unG-8S-bvw">
                                                         <rect key="frame" x="4" y="4.3333333333333339" width="17" height="17"/>
                                                         <color key="tintColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                                                        <imageReference key="image" image="cube.box.fill" catalog="system" symbolScale="default"/>
+                                                        <imageReference key="image" image="shippingbox.fill" catalog="system" symbolScale="default"/>
```
Comment (victor-sarda) [inline ViteMaDose/Views/CentresList/Cells/CentreCell.xib:160]: `shippingbox` is only available with iOS 14, lets leave `cube.box` for now and ignore the warning 👍 
```suggestion
                                                        <imageReference key="image" image="cube.box.fill" catalog="system" symbolScale="default"/>
```

**Review by victor-sarda — CHANGES_REQUESTED** [review]
Looks all good to me except the warning thing, I left a change request. Happy to merge after this, thanks 👍

File: ViteMaDose/Views/CentresList/Cells/CentreCell.xib (hunk 1 above)
Comment (yageek) [reply ViteMaDose/Views/CentresList/Cells/CentreCell.xib:160]: My bad 🤦‍♂️

File: ViteMaDose/Views/CentresList/Cells/CentreCell.xib [hunk 2]
```diff
@@ -253,9 +253,9 @@
     </objects>
     <resources>
         <image name="calendar" catalog="system" width="128" height="106"/>
-        <image name="cube.box.fill" catalog="system" width="128" height="124"/>
         <image name="location.fill" catalog="system" width="128" height="121"/>
         <image name="phone.fill" catalog="system" width="128" height="114"/>
+        <image name="shippingbox.fill" catalog="system" width="128" height="124"/>
```
Comment (victor-sarda) [inline ViteMaDose/Views/CentresList/Cells/CentreCell.xib:258]: Same here sorry 😅

Comment (victor-sarda) [inline ViteMaDose/Views/CentresList/Cells/CentreCell.xib:258]:
```suggestion
        <image name="cube.box.fill" catalog="system" width="128" height="124"/>
```

**Review by victor-sarda — APPROVED** [review]

**Review by NathanFallet — APPROVED** [review]
👌

**Pull request merged**